langchain/utilities/powerbi.py

```diff
@@ -16,10 +16,6 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-if TYPE_CHECKING:
```

Comment:
why remove this?

Reply:
(this only affects linting, won't cause any runtime issues)

Reply:
+1

Reply:
fixed by using a different typing class from azure.core, still needs to be regular import in order to get through validations at runtime.